src/renderer/src/pages/Settings.tsx

```diff
@@ -12,8 +12,15 @@ export function Settings() {
             window.api.store.set('openai-key', value)
           }} size="sm" label="OpenAI key" />
         </div>
+
+        <div className="mt-5">
+        <Input spellCheck={false} defaultValue={window.api.store.get('openai-base-url') as string} onChange={e => {
```

Comment:
add a placeholder to the default base URL

Reply:
added.